dask/delayed.py

```diff
@@ -521,6 +521,12 @@ def __dir__(self):
     def __getattr__(self, attr):
         if attr.startswith("_"):
             raise AttributeError("Attribute {0} not found".format(attr))
+
+        if attr == 'visualise':
+            # added to warn users incase of spelling error 
+            # for more details: https://github.com/dask/dask/issues/5721
+            raise AttributeError("Attribute visualise not found. Perhaps you meant visualize?")
```

Comment:
We shouldn't raise an error here, we should warn: https://docs.python.org/3/library/warnings.html.

```suggestion
            warnings.warn(
                "dask.delayed objects have no `visualise` method, perhaps you meant `visualize`?"
            )
```

This will warn users that may have accidentally spelled it `visualise`, but won't error, so users that may intentionally be calling `visualise` (perhaps they've delayed an object that has such a method) won't have their code broken.

Reply:
sure.  Thanks @jcrist